src/testSchemas/arraySchema.tsx

```diff
@@ -10,8 +15,29 @@ export const arrayControlUISchema = {
   type: "VerticalLayout",
   elements: [
     {
+      type: "Control",
       scope: "#/properties/assets",
+    },
+  ],
+} satisfies UISchema<typeof objectArrayControlJsonSchema>
+
+export const arrayControlSortableUISchema = {
+  type: "VerticalLayout",
+  elements: [
+    {
       type: "Control",
+      scope: "#/properties/assets",
+      options: {
+        showSortButtons: true,
+        moveUpButtonProps: {
+          icon: <ArrowUpOutlined />,
+          onClick: () => {}, // User should be unable to override the onClick event
```

Comment:
This only works because we are spreading these props before passing the `onClick` handler correct? Could you add a test so there can't be a regression on this?

Reply:
I copied this from the `arrayControlUISchemaWithIcons` schema but I'll add the test to cover it

Reply:
In answer to your first question- you are correct.  I was apparently testing the onClicks couldn't be overridden without realizing it, but i added an additional test to call that out.  I also added an additional story/test so that the array up/down buttons will display labels by default unless an icon is provided by the UISchema.